### PR TITLE
Add Eclipse Java shortcut

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

**Comment by mimi89999** [comment]
This doesn't cause issues with stable `just-install` releases, so I think that it could be merged.

File: just-install-v4.schema.json [hunk 1]
```diff
@@ -39,6 +39,9 @@
         },
         "shims": {
           "type": "array"
+        },
+        "shortcuts": {
+          "type": "array"
```
Comment (lvillani) [inline just-install-v4.schema.json]: Should add structural information about objects in the array, something like:

```suggestion
          "type": "array",
          "items": {
            "type": "object",
            "additionalProperties": false,
            "required": ["name", "target"],
            "properties": {
              "name": {
                "type": "string"
              },
              "target": {
                "type": "string"
              }
            }
          }
```

File: just-install-v4.json [hunk 2]
```diff
@@ -665,7 +665,11 @@
       "installer": {
         "kind": "zip",
         "options": {
-          "destination": "{{.PROGRAMFILES}}\\Eclipse-Java"
+          "destination": "{{.PROGRAMFILES}}\\Eclipse-Java",
+          "shortcuts": [{
+              "name": "Eclipse Java",
```
Comment (lvillani) [inline just-install-v4.json]: Nitpick: indent using two spaces instead of four

**Comment by lvillani** [comment]
Looks good, thanks! 👍

**Pull request merged**